pegasus/sites.v3/hourofcode.com/public/events/2013/index.haml

```diff
@@ -13,12 +13,17 @@ nav: events_nav
   state_column = Forms.json('data.state_code_s')
   events = Forms.events_by_country(kind, country_column: country_column)
   us_events = Forms.events_by_state(kind, country_column: country_column, state_column: state_column)
+  us_events_count = us_events.inject(0) do |result, element|
+    result + element[:count]
+  rescue
+    0
+  end
 
 %h1= hoc_s(:events_all_old_title).gsub(/\@year/, hoc_year.to_s)
 
 %ul
   %li{class: "hoc-event-country"}
-    %span{style: "font-weight: bold;"}= "United States (#{us_events.count rescue '0'} #{hoc_s(:events)})"
+    %span{style: "font-weight: bold;"}= "United States (#{us_events_count} #{hoc_s(:events)})"
```

Comment:
Note that this is actually a *slight* refactoring of this very old (and likely entirely unused) page, to better align with how newer pages work. Because `us_events` is a group of collections, simply calling `.count` on it will return the number of collections in the group, not the number of events in all the collections.